charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -65,6 +65,8 @@ defaultDashboards:
 
 # -- Create default rules for monitoring the cluster
 defaultRules:
+  # -- Labels, which are additionally to cluster are added to group expr
+  exprLabels: []
```

Comment:
```suggestion
  # -- Labels, which are used for groupping results of the queries. Note that these labels are joined with `.Values.global.clusterLabel`
  additionalGroupByLabels: []
```

Reply:
Slightly rephrased comment and the name of parameter to make it easier to understand by users. LGTM other than that